src/Pages/Main/Main.scss

```
@@ -0,0 +1,216 @@
+          border-bottom-left-radius: 4px;
+          color: #737373;
+
+          &:focus {
+            border-color: #48ffe8;
+            outline: 1px;
+            box-shadow: 0 0 0 3px rgba(0, 199, 174, 0.5);
+          }
+        }
+
+        .btn-main {
+          width: 20%;
+          height: 54px;
+          padding: 14px 24px;
+          border-top-right-radius: 4px;
+          border-bottom-right-radius: 4px;
+          background-color: var(--primary);
+          color: var(--white);
+          font-size: 16px;
+          font-weight: 550;
+
+          &::before {
+            content: "";
+            color: #fff;
+          }
+
+          &:hover {
+            background-color: #00a18d;
+          }
+        }
+      }
+    }
+  }
+
+  .main-nav {
+    background-color: var(--light);
+
+    .main-nav-category {
+      display: flex;
+      justify-content: space-around;
+      max-width: 1200px;
+      margin: 0 auto;
+
+      .main-nav-category-li {
+        width: 12.5%;
+        height: 104px;
+        padding: 6px 8px;
+        cursor: pointer;
+
+        .cate-wrap {
+          padding: 8px 0;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+
+          &:hover {
+            padding-top: 0;
+
+            img {
+              padding-bottom: 8px;
+            }
+          }
+
+          p {
+            line-height: 24px;
+            font-size: 14px;
+            color: #323232;
+            text-align: center;
+          }
+        }
+      }
+    }
+  }
+
+  .main-popular-service {
+    padding: 50px 0;
+
+    .main-popular-service-h2 {
+      margin-bottom: 50px;
+      font-size: 32px;
+      font-weight: 500;
+      text-align: center;
+      line-height: 48px;
+      color: var(--dark);
+    }
+
+    .popular-card-wrap {
+      max-width: 1200px;
+      margin: 0 auto;
+      display: flex;
+      flex-wrap: wrap;
+
+      .popular-card {
+        width: (100% / 3);
+        padding: 0 15px 15px 15px;
+
+        .card-outline {
+          border-radius: 4px;
+          box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+
+          .popular-card-img {
+            background-size: cover;
+            background-position: 50% 50%;
+            height: 150px;
+            border-top-left-radius: 4px;
+            border-top-right-radius: 4px;
+          }
+
+          .popular-card-title {
+            line-height: 33px;
+            padding: 16px 0;
+            font-size: 16px;
+            text-align: center;
+            color: var(--dark);
+          }
+        }
+      }
+    }
+  }
+
+  .main-whole-service {
+    padding: 50px 0;
+
+    .main-whole-service-2 {
+      margin-bottom: 50px;
+      font-size: 32px;
+      font-weight: 500;
+      text-align: center;
+      line-height: 48px;
+      color: var(--dark);
+    }
+
+    .whole-category {
+      column-count: 3;
+      max-width: 1200px;
+      margin: 0 auto;
+      padding: 0 15px;
+
+      .whole-category-li {
```

```suggestion
    .whole-category {
      column-count: 3;
      max-width: 1200px;
      margin: 0 auto;
      padding: 0 15px;

      &-li {
```
이렇게 바꿔주면 클래스 명도 간결하게 줄여 쓸 수 있습니다.